build.sbt

```diff
@@ -37,11 +37,6 @@ scalacOptions ++= Seq(
   , "-Xlint:_,-nullary-unit"  // Enable or disable specific warnings (see list below)
   , "-Yno-adapted-args"       // Do not adapt an argument list to match the receiver
   , "-Ywarn-dead-code"        // Warn when dead code is identified
-// Not really useful
-//  , "-Ywarn-numeric-widen"    // Warn when numerics are widened
-  , "-Ywarn-unused"           // Warn when local and private vals, vars, defs, and types are are unused
-  , "-Ywarn-unused-import"    // Warn when imports are unused
-  , "-Ywarn-value-discard"    // Warn when non-Unit expression results are unused
```

Comment:
Any reason to have removed these three compiler options? Did they trigger any warnings you could not fix? Please, bear in mind that this should affect only the tests, not REPLesent usage itself.